packages/components/src/components/data-list/data-list.tsx

```diff
@@ -149,6 +158,7 @@ const DataList = React.memo(
         };
 
         const handleScroll = (ev: Partial<React.UIEvent<HTMLDivElement>>) => {
+            if (scroll_top !== 0) clearScroll();
```

Comment:
here you can check if clearScroll was passed, then run the function

Reply:
but i think clearscroll wont go undefined since its given a default value